openpype/hosts/maya/plugins/publish/validate_render_image_rule.py

```diff
@@ -24,8 +26,12 @@ class ValidateRenderImageRule(pyblish.api.InstancePlugin):
 
     def process(self, instance):
 
-        required_images_rule = self.get_default_render_image_folder(instance)
-        current_images_rule = cmds.workspace(fileRuleEntry="images")
+        required_images_rule = os.path.normpath(
+            self.get_default_render_image_folder(self, instance)
```

Comment:
```suggestion
            self.get_default_render_image_folder(instance)
```
it may error out with (self, instance).